Automattic/tour
# Tour block

This PR adds a Gutenberg block with the tours available.

### Frontend:

![image](https://github.com/Automattic/tour/assets/1667814/5975f5ee-54a7-4e5b-98ba-d942e37e6334)

### Backend:

![image](https://github.com/Automattic/tour/assets/1667814/e3f81fb0-d0e9-4529-a417-d4cafa85fd1a)

## Test

To test this PR, you can add a new "Tour Block" to a page or a post, and you will see a list with the current tours. Click in one of these tours to start showing it. This PR always try to show the tour: it doesn't check if the selected tour is available in the current URL.

If you want to build the block, you need to:
- Install the dependencies, running `npm install`
- Build the block, running `npm run build`
